lib/cs_guide/resources/venue.ex

```diff
@@ -9,8 +9,14 @@ defmodule CsGuide.Resources.Venue do
 
   schema "venues" do
     field(:venue_name, :string)
+    field(:description, :string)
+    field(:address1, :string)
+    field(:address2, :string)
```

Comment:
we don't _need_ an **`:address2`**. Google's Material Design pattern https://material.io uses a _single_ field for address, see: https://github.com/dwyl/fields/issues/21 ... 
so we don't _need_ to overcomplicate this.
please remove. ✂️ 
thanks! ✨